src/plugins/visualizations/public/legacy/build_pipeline.ts

```diff
@@ -155,7 +158,8 @@ export const getSchemas = (vis: Vis, { timeRange, timefilter }: BuildPipelinePar
       }
     }
     if (schemaName === 'split') {
-      schemaName = `split_${vis.params.row ? 'row' : 'column'}`;
+      // TODO: We should check if there's a better way then casting to `any` here
+      schemaName = `split_${(vis.params as any).row ? 'row' : 'column'}`;
```

Comment:
We could restrict `TVisParams` a little further (`<TVisParams extends { row?: boolean }>`), but this file is not properly typed in a lot of places so I wouldn't bother improving it. We should be able to remove it and move the logic into the individual visualizations where they can be typed explicitly anyway.